extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -612,8 +612,6 @@ public void startElement(String uri, String localName, String qName, Attributes
                 } else {
                     textField = null;
                 }
-            } else if (qName.equals("logo")) {
-                // logo = attributes.getValue("UUID");
```

Comment:
Do you know why this instruction has been removed?

Reply:
No, but it hasn't been modified for 2 years and 5 months, so I thought that it is not used.

Reply:
```
commit 9a2fc9cba501faa943f2e4aee3ce30ee67f6f77f (origin/pull/548)
Author: Karel Goderis <karel.goderis@me.com>
Commit: Karel Goderis <karel.goderis@me.com>

    Migration of the Sonos 2.0 binding from openHAB 2 to Eclipse Smarthome
    
    Also-by: Andre Fuechsel <a.fuechsel@telekom.de>
    Also-by: Dan Cunningham <dan@digitaldan.com>
    Also-by: David Wheeler <hippysoyboy@gmail.com>
    Signed-off-by: Karel Goderis <karel.goderis@me.com>
```

@kgoderis It seems the line has been comment out as it has been added to the ESH repo.

Do you know why it is present but comment out?

Reply:
@maggu2810 Nope - I can't remember doing it and for what reason. I think the UUID refers to some bitamp that is internally stored on the Sonos player, and since we do not use that in our own GUI's, I think it can be removed, and as @kssimova pointed out, nobody complained since 2.5 years....

Reply:
We can comment out the whole branch in case someday we want to use it?

Reply:
Feel free to delete it completely. There is no need to keep commented-out code "in stock" 😄